Lib/test/test_free_threading/test_dict.py

```diff
@@ -10,9 +10,10 @@
 
 from _testcapi import dict_version
 
-from test.support import threading_helper
+from test.support import requires_resource, threading_helper
 
 
+@requires_resource('cpu')
```

Comment:
Would you please skip this outside of the free-threaded build (i.e., `not support.Py_GIL_DISABLED`) instead of marking this as cpu-only?

Reply:
Maybe skip `test_freethreading` in its entirety? #120660